### ISSUE TITLE
Remove user and group

### ISSUE THREAD

**Issue by apachemaven** [takezoe/gitbucket#599]
Is it possible to remove specified user and group? I can not find the entry point.

For the user, one can login and delete the account itself. 

I wonder if this can be done by the administrator.

**Comment by dontbufferoverflowme** [comment]
The administrators are allowed to create,edit and delete groups and users as far as I see it within my gitbucket installation.

**Comment by apachemaven** [comment]
But I can not find the entry point: http://imgbin.org/images/21780.png

Did I miss anything?

**Comment by dontbufferoverflowme** [comment]
Actually I missed something, sry. I just checked it more thoroughly. 
As admin you can manage groups and delete them. For the user you have only the "disable" checkmark inside edit menu. When you have done it the particular user will stop appear in the "User Management" list and (I assume) can't login. In this menu you see " Include removed users" checkmark which will show current and disabled users. 
I checked it with the 2.7 version. I remember the earlier versions had "delete"-Button for users which has the exact same effect as the disable checkmark. About the last sentence I am not sure, pls correct me if I'm wrong.

**Comment by apachemaven** [comment]
Yes, that's right, I am using 2.7 too, and the logic of the management is exactly same as you said. 

I am not sure for the previous version.

**Comment by Crayon2000** [comment]
I **think** this is important for General Data Protection Regulation ([GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation)) compliance.

To conform to these:
* You automatically delete data that your business no longer has any use for ([Article 5 – Principles relating to processing of personal data](https://advisera.com/eugdpracademy/gdpr/principles-relating-to-processing-of-personal-data))
* Your customers can easily request deletion of their personal data ([Article 17 – Right to erasure](https://advisera.com/eugdpracademy/gdpr/right-to-erasure-right-to-be-forgotten))


PS: I have limited knowledge of GDPR :smile:

**Comment by shinsenter** [comment]
+1 for user deletion.
